hardhat.config.ts

```diff
@@ -25,19 +28,29 @@ module.exports = {
       url: "http://127.0.0.1:8545",
     },
     mainnet: {
-      url: `https://mainnet.infura.io/v3/${INFURA_API_KEY}`,
+      url: "https://rinkeby.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
```

Comment:
@drinkius Is this your Infura api key? Even though it's for Rinkeby, it's better to have this removed

Reply:
Not mine, publicly available in databases online. Added it this way to remove hassle for everyone 

Source:
https://rpc.info/